src/client/game.cpp

```diff
@@ -4341,15 +4345,17 @@ void the_game(bool *kill,
 		}
 
 	} catch (SerializationError &e) {
-		error_message = std::string("A serialization error occurred:\n")
-				+ e.what() + "\n\nThe server is probably "
-				" running a different version of " PROJECT_NAME_C ".";
+		const char *s = gettext("\n\nThe server is probably running a different version of %s.");
+		char ver_err[128];
+		porting::mt_snprintf(ver_err, sizeof(ver_err), s, PROJECT_NAME_C);
+		error_message = strgettext("A serialization error occurred:\n")
+				+ e.what() + ver_err;
```

Comment:
minor adjustment: keeping the newlines out of the translated string
```suggestion
		const char *s = gettext("The server is probably running a different version of %s.");
		char ver_err[128];
		porting::mt_snprintf(ver_err, sizeof(ver_err), s, PROJECT_NAME_C);
		error_message = strgettext("A serialization error occurred:") + "\n"
				+ e.what() + "\n\n" + ver_err;
```